templates/observability/managed.html

```diff
@@ -0,0 +1,577 @@
+          }}
+        </div>
+        <div class="p-logo-section__item">
+          {{ 
+            image (
+            url="https://assets.ubuntu.com/v1/cf962284-docapost.png",
+            alt="",
+            width="160",
+            height="120",
+            hi_def=True,
+            loading="lazy",
+            attrs={"class": "p-logo-section__logo"}
+            ) | safe
+          }}
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Cloud-native applications need cloud-native observability</h2>
+      <p>One practitioner out of five thinks that observability is a key challenge for cloud native environments. One in two practitioners says that lack of manpower is the worst inhibitor to their cloud journey. With todays many components of the cloud native ecosystem, engineers spend a lot of time and effort setting up and maintaining their monitoring tools. Let's change that!</p>
+      <p>
+        <a href="/observability/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+      </p>
+    </div>
+    <div class="col-4 u-vertically-center">
+      <div class="p-card">
+        <div class="p-heading-icon--small">
+          <div class="p-heading-icon__header">
+            {{ 
+              image (
+              url="https://assets.ubuntu.com/v1/b061c401-White+paper.svg",
+              alt="",
+              width="171",
+              height="150",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+            }}
+            <p>Industry report</p>
+          </div>
+        </div>
+        <p class="p-card__content">
+          <a href="https://juju.is/cloud-native-kubernetes-usage-report-2021" class="p-link--external">The Kubernetes and cloud native operations report 2021</a>
+        </p>
+      </div>
+    </div>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr>
+</div>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h2>Never miss another outage</h2>
+      <p>Your monitoring systems must be up and running to tell you what else is not. The availability of your business applications is effectively capped by the availability of the systems monitoring them. Get the best OSS monitoring tools run with spectacular uptime for you, by Canonical.</p>
+      <p>
+        <a href="/observability/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+      </p>
+    </div>
+    <div class="col-4 u-vertically-center u-align--center u-hide--small">
+      {{ 
+        image (
+        url="https://assets.ubuntu.com/v1/50dea393-fixes+in+24hrs.svg",
+        alt="",
+        width="174",
+        height="180",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<div class="u-fixed-width">
+  <hr>
+</div>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
```

Comment:
```suggestion
  <div class="u-fixed-width">
```
is the same as row<col-12